swift-sdk/IterableConfig.swift

```diff
@@ -55,6 +55,13 @@ import Foundation
 /// The delegate for getting the authentication token
 @objc public protocol IterableAuthDelegate: AnyObject {
     @objc func onAuthTokenRequested(completion: @escaping AuthTokenRetrievalHandler)
+    @objc func onTokenRegistrationFailed()
+}
+
+extension IterableAuthDelegate {
+    func onTokenRegistrationFailed() {
```

Comment:
Does it need to have a empty delegate function inside config?

Reply:
I'll move this to inside `AuthManager`.